PNChart/PNBarChart.h

```diff
@@ -10,10 +10,6 @@
 #import "PNChartDelegate.h"
 #import "PNBar.h"
 
-#define xLabelMargin 15
-#define yLabelMargin 15
```

Comment:
This is a fork with different features than the original repo. Updating it to latest version is not straight forward, I'm just fixing our errors so the pod compiles in universal.